### DISCUSSION TITLE
Is it possible to launch a sandbox's Run Menu item via commandline?

### DISCUSSION THREAD

**Discussion by TFWol** [sandboxie-plus/Sandboxie#4076] [Q&A Feedback]
For example if I wanted to run "Raft Mod Launcher":
![image](https://github.com/user-attachments/assets/5ab3a366-a59b-4cc3-9fb7-51e1c655ac69)

Could I do something like:
`start.exe /box:Steam /run_menu:"Raft Mod Launcher"`

**Comment by edions** [comment]
I don't think there's a direct way to do that yet.

One method I know that is quite similar to what you want involves specifying the full path to the executable.
`Start.exe /box:Steam "C:\Program Files\RaftModLoader\RaftModLoader.exe"`

Some programs also work without specifying the full path.
`Start.exe /box:Browser firefox.exe`

Here's the old documentation for all possible start commands.
https://sandboxie-website-archive.github.io/www.sandboxie.com/StartCommandLine.html

**Reply by TFWol** [reply]
Yeah, that's been my method for quite a while (and still is).
I was hoping to just have the Run menu to worry about; programs don't seem to run exactly as expected using start.exe vs picking from Run.

**Reply by edions** [reply]
Yeah, but unfortunately, I don't think there's another way to do it though.
Facing the exact same problem with programs installed inside sandbox, usually I need to type the full path for it to work. Something like this: `Start.exe /box:Browser "C:\Sandbox\Desktop\Browser\user\all\Mozilla\firefox.exe"`
But yeah, I hope there's a better way to do this.
